source/marc/terms.ttl

```diff
@@ -26,6 +26,7 @@ marc:relatedAs a owl:ObjectProperty ;
     rdfs:range kbv:ObjectProperty .
 
 marc:electronicLocatorType a owl:DatatypeProperty ;
+    sdo:domainIncludes kbv:Document, kbv:MediaObject, kbv:Electronic ;
```

Comment:
If you add: `:MediaObject rdfs:subClassOf :Document` where `:MediaObject` is declared (relations.ttl AFAIK) and verify that :Electronic is a subclass of document (via :Instance), we can use just `rdfs:range kbv:Document` throughout in this change.

Reply:
Good idea. I'll fix.